scripts/runningnotes_from_sequencing.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+DESC = """EPP script to summarize sequencing start results to the projects running notes
+"""
+import datetime
+import re
+import xml.etree.ElementTree as ET
+from argparse import ArgumentParser
+
+from genologics.config import BASEURI, PASSWORD, USERNAME
+from genologics.entities import Process
+from genologics.lims import Lims
+from write_notes_to_couchdb import write_note_to_couch
+
+from scilifelab_epps.wrapper import epp_decorator
+
+regex_project_line = re.compile(r"^\[(P\d+)\][\s]*:")
+TIMESTAMP: str = datetime.datetime.now().strftime("%y%m%d_%H%M%S")
+
+
+@epp_decorator(script_path=__file__, timestamp=TIMESTAMP)
+def main(args):
+    lims = Lims(BASEURI, USERNAME, PASSWORD)
+    pro = Process(lims, id=args.pid)
+    pro_udfs = {}
+    samples = {}
+    general_comments = []
+    project_specific_comments = {}
+
+    for name, value in pro.udf.items():
+        pro_udfs[name] = value
+    seq_setup = f"{pro_udfs.get('Read 1 Cycles')}_{pro_udfs.get('Index Read 1')}_{pro_udfs.get('Index Read 2')}_{pro_udfs.get('Read 2 Cycles')}"
+    inst = f"{pro_udfs.get('Instrument')} {pro_udfs.get('Side')}"
+
+    for line in pro_udfs.get("Comment").splitlines():
+        if not line.startswith("//"):
```

Comment:
Are these lines comments inside the comments?

Reply:
They are instructions for filling in the comments that will be entered as default in the field in LIMS.

Reply:
It would be helpful to have some in-line comments in the code explaining this

Reply:
Wrote some, let me know if it needs more

Reply:
Looks good!